ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -385,3 +390,112 @@ def set_upgrade_images(self):
             with NamedTemporaryFile() as cs_yaml:
                 dump_data_to_temp_yaml(cs_data, cs_yaml.name)
                 ocs_catalog.apply(cs_yaml.name)
+
+
+class UpgradeDisruptiveOps(EcosystemTest):
```

Comment:
There is currently no usage of class as no actual reference to self is really needed here.  Maybe in the future you are going to use it but for now I see that two bellow can just live in upgrade module and as you really do not store any object member which needs to be shared it will simplify the logic here.  Or you can create those class as class methods and people can use them without creating an instance.

Reply:
as discussed, i will move it to another module for disruptive operations, and no class will be needed to gather them all in same place.